README.md

```diff
@@ -35,6 +35,12 @@ Production database is configured in a similar fashion, but the project is not r
 
 ## Development setup
 
+### Using Docker
+This was tested using v1.12.6 of [Docker](https://docs.docker.com/engine/installation/linux/ubuntu/) and v1.11.2 of [Docker Compose](https://docs.docker.com/compose/install/). Any version below these are untested.
+To deploy this application in a docker instance go to the root of this repository and run `docker-compose up`. This will run through the configurations
```

Comment:
Will you please add something here about stopping any local mysql process and be sure to wait until `docker-compose up` finishes? I think that is what messed up my first try.